### PR TITLE
Update examples

### PR DESCRIPTION
**Pull request by DylanC**
Update package deps for the examples. Only issue remaining is "Premature close" on many of the examples.

Added readme to explain usage of Proxy examples.

### CODE REVIEW

**Comment by goto-bus-stop** [comment]
I get
```
Error: Missing `pino-pretty` module: `pino-pretty` must be installed separately
```
on the server-side-rendering example. do you see the same?

Pino is a logging module used by fastify. Likely that update requires that you also add a separate dependency on pino-pretty.

**Comment by DylanC** [comment]
What command did you run to get this?

**Comment by goto-bus-stop** [comment]
```
clinic bubbleprof --on-port 'node server-side-rendering/stories.js' -- node server-side-rendering/server
```
(though the `--on-port` flag isn't the thing that crashes for me, just `clinic bubble -- node server-side-rendering/server` has the same effect.)

**Comment by DylanC** [comment]
I can see that issue now. I've added `pino-pretty` to the package.

File: server-side-rendering/package.json [hunk 1]
```diff
@@ -8,22 +8,22 @@
...
-    "react-dom": "^16.4.1",
-    "react-router-dom": "^4.3.1",
-    "rss-parser": "^3.4.2",
-    "sanitize-html": "^1.18.2",
-    "typestyle": "^2.0.1"
+    "node-fetch": "^2.6.0",
+    "pino-pretty": "^3.6.1",
+    "react": "^16.13.1",
+    "react-dom": "^16.13.1",
+    "react-router-dom": "^5.1.2",
```
Comment (goto-bus-stop) [inline server-side-rendering/package.json]: I think this update may require code changes, we could just pin it to 4.x for now.

In terminal 1:
```
node server-side-rendering/server-for-data
```
In terminal 2:
```
clinic bubbleprof --on-port 'node server-side-rendering/stories' -- node server-side-rendering/server
```
throws a whole bunch of errors atm :)

Comment (DylanC) [reply server-side-rendering/package.json]: Yeah I was worried about the large jump in version numbers. :laughing: Looking into the errors at the moment.

**Pull request merged**